webapp/javascript/redux/reducers/settings.ts

```diff
@@ -0,0 +1,63 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import { Users } from '@models/users';
+import { fetchUsers } from '@pyroscope/services/users';
+import type { RootState } from '../store';
+import { addNotification } from './notifications';
+
+interface SettingsRootState {
+  // Since the value populated from the server
+  // There's no 'loading'
+  users:
+    | { type: 'loaded'; data: Users }
+    | { type: 'reloading'; data: Users }
+    | { type: 'failed'; data: Users };
+}
+
+// Define the initial state using that type
+const initialState: SettingsRootState = {
+  users: { type: 'loaded', data: (window as any).initialState.users },
```

Comment:
Wait is this populated from the server?

Reply:
Actually It looks like I left something uncommitted